portfolio/src/main/java/com/google/sps/data/ShortestTaskFirstScheduler.java

```diff
@@ -40,9 +40,9 @@
     List<CalendarEvent> eventsList = new ArrayList<CalendarEvent>(events);
     List<Task> tasksList = new ArrayList<Task>(tasks);
     Collections.sort(tasksList, sortByTaskDurationThenName);
-    CalendarGroup calendarGroup =
-        new CalendarGroup(eventsList, workHoursStartTime, workHoursEndTime);
-    List<TimeRange> availableTimes = calendarGroup.calculateFreeTimeRanges();
+    CalendarEventsGroup CalendarEventsGroup =
```

Comment:
Variable name should start with lowercase rather than uppercase letter.

CalendarEventsGroup CalendarEventsGroup => CalendarEventsGroup calendarEventsGroup

Reply:
Updated.